db/migrate/20191120152618_add_parent_to_legislations.rb

```diff
@@ -0,0 +1,5 @@
+class AddParentToLegislations < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :legislations, :parent, null: true
```

Comment:
I think it should be
```
add_reference :legislations, :parent, foreign_key: { to_table: :legislations }, index: true, null: true
```

not sure if we need index

Reply:
but we need foreign key

Reply:
I'm doing this in a middle of another chat now, so sorry for this. Maybe I should squash some of these commits to avoid messes for other people.

Reply:
no need to squash I think